<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>551</width>
    <height>576</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>QuickShell</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../assets.qrc">
    <normaloff>:/images/logo.png</normaloff>:/images/logo.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>528</width>
      <height>506</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="1">
     <property name="spacing">
      <number>8</number>
     </property>
     <property name="sizeConstraint">
      <enum>QLayout::SizeConstraint::SetDefaultConstraint</enum>
     </property>
     <property name="leftMargin">
      <number>8</number>
     </property>
     <property name="topMargin">
      <number>8</number>
     </property>
     <property name="rightMargin">
      <number>8</number>
     </property>
     <property name="bottomMargin">
      <number>8</number>
     </property>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Choose one script:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSplitter" name="splitter">
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
         </property>
         <widget class="QWidget" name="verticalLayoutWidget_2">
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0">
             <item>
              <widget class="QSplitter" name="splitter_2">
               <property name="orientation">
                <enum>Qt::Orientation::Horizontal</enum>
               </property>
               <property name="handleWidth">
                <number>4</number>
               </property>
               <widget class="QListWidget" name="listCategory">
                <property name="contextMenuPolicy">
                 <enum>Qt::ContextMenuPolicy::CustomContextMenu</enum>
                </property>
               </widget>
               <widget class="QListWidget" name="listScripts">
                <property name="contextMenuPolicy">
                 <enum>Qt::ContextMenuPolicy::CustomContextMenu</enum>
                </property>
               </widget>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
         <widget class="QTextBrowser" name="plainTextDetails">
          <property name="markdown">
           <string>Please select any script to display detailed information.

</string>
          </property>
          <property name="openLinks">
           <bool>false</bool>
          </property>
         </widget>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Execution Arguments:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_ExecutionArguments"/>
         </item>
         <item>
          <widget class="QPushButton" name="btnRunScript">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Run the selected script</string>
           </property>
           <property name="icon">
            <iconset theme="QIcon::ThemeIcon::MediaPlaybackStart"/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>551</width>
     <height>27</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_P">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_Import"/>
    <addaction name="separator"/>
    <addaction name="separator"/>
    <addaction name="action_Options"/>
    <addaction name="separator"/>
    <addaction name="action_Exit"/>
   </widget>
   <widget class="QMenu" name="menu_H">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="action_README"/>
    <addaction name="separator"/>
    <addaction name="action_Qt"/>
    <addaction name="action_About"/>
   </widget>
   <widget class="QMenu" name="menu_View">
    <property name="title">
     <string>&amp;View</string>
    </property>
    <addaction name="action_Refresh"/>
    <addaction name="action_ReloadCollection"/>
   </widget>
   <addaction name="menu_P"/>
   <addaction name="menu_View"/>
   <addaction name="menu_H"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_Options">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::DocumentProperties"/>
   </property>
   <property name="text">
    <string>&amp;Preference</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="action_Exit">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::ApplicationExit"/>
   </property>
   <property name="text">
    <string>E&amp;xit</string>
   </property>
  </action>
  <action name="action_README">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::HelpFaq"/>
   </property>
   <property name="text">
    <string>See &amp;README</string>
   </property>
  </action>
  <action name="action_About">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::HelpAbout"/>
   </property>
   <property name="text">
    <string>&amp;About QuickShell</string>
   </property>
  </action>
  <action name="action_Qt">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::DialogInformation"/>
   </property>
   <property name="text">
    <string>About &amp;Qt</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::TextHeuristicRole</enum>
   </property>
  </action>
  <action name="action_Import">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::DocumentOpen"/>
   </property>
   <property name="text">
    <string>&amp;Import</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
  <action name="action_Refresh">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::ViewRefresh"/>
   </property>
   <property name="text">
    <string>&amp;Refresh</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
  <action name="action_ReloadCollection">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::ViewRefresh"/>
   </property>
   <property name="text">
    <string>Reload &amp;Collection  </string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F5</string>
   </property>
  </action>
  <action name="actionRunSelectedScript">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::MediaPlaybackStart"/>
   </property>
   <property name="text">
    <string>Run the selected script</string>
   </property>
   <property name="toolTip">
    <string>Run the selected script</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::TextHeuristicRole</enum>
   </property>
  </action>
  <action name="action_EditScriptFile">
   <property name="icon">
    <iconset resource="../assets.qrc">
     <normaloff>:/images/assets/icons/text-editor.png</normaloff>:/images/assets/icons/text-editor.png</iconset>
   </property>
   <property name="text">
    <string>Edit the Selected Script File</string>
   </property>
   <property name="toolTip">
    <string>Edit the Selected Script File</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::TextHeuristicRole</enum>
   </property>
  </action>
  <action name="action_RunTheSelectedScriptsNotIncludeArguments">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::MediaPlaybackStart"/>
   </property>
   <property name="text">
    <string>Run the selected scripts (not include arguments)</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::TextHeuristicRole</enum>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../assets.qrc"/>
 </resources>
 <connections/>
</ui>
